src/engine/sparqlExpressions/PrefilterExpressionIndex.h

```diff
@@ -264,7 +264,7 @@ void checkPropertiesForPrefilterConstruction(
 template <CompOp comparison>
 std::vector<PrefilterExprVariablePair> makePrefilterExpressionVec(
     const IdOrLocalVocabEntry& referenceValue, const Variable& variable,
-    bool mirrored);
+    bool mirrored, bool prefilterDate = false);
```

Comment:
The special case for the `prefilterDate` should also be documented, and maybe the argument name is better `prefilterDateByYear`.